LICENSE-3rdparty.csv

```diff
@@ -1,4 +1,7 @@
 Component,Origin,License,Copyright
+require,@adonisjs/fold,MIT,Copyright 2019 Harminder Virk 
+require,@adonisjs/framework,MIT,Copyright 2019 Harminder Virk 
```

Comment:
to be installed via `versions`

Reply:
Should I take the lines out? It was failing the licensing tests without these lines.

Reply:
You added `@adonisjs/framework` as a dev dependency.  When you remove it and use `versions` in the test the check should pass.  seems like fold and sink are fine afaict.

Reply:
removing the `@adonisjs/framework` causes it to fail the lint